### PR TITLE
Feature/hardware select custom path

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
There was a missed use case of the Derivation Path select allowing text input for a custom path
This PR adds that dynamic select/input and toggle back and for upon select

### CODE REVIEW

**Review by Pojielp — APPROVED** [review]

**Review by Pojielp — APPROVED** [review]

File: packages/common/src/views/AccountSelect.svelte [hunk 1]
```diff
@@ -263,18 +316,30 @@
...
+            <input type='text' 
+              class='base-path-select' 
+              placeholder="type/your/custom/path..." 
+              on:change={e => handleCustomPath(e)}
+            /> 
+            <span class='input-select' on:click={toggleDerivationPathToDropdown}></span>
+          {:else if  !customDerivationPath}
+            <select
+              class='base-path-select'
+              on:change={(e) => handleDerivationPathSelect(e)}
```
Comment (taylorjdawson) [inline packages/common/src/views/AccountSelect.svelte:329]:
```suggestion
              on:change={handleDerivationPathSelect}
```

File: packages/common/src/views/AccountSelect.svelte [hunk 2]
```diff
@@ -263,18 +316,30 @@
...
-            {#each basePaths as path, pathIndex}
-              <option
-                value={path.value}
-              >
-                {path.label} - {path.value}
+          {#if customDerivationPath}
+            <input type='text' 
+              class='base-path-select' 
+              placeholder="type/your/custom/path..." 
+              on:change={e => handleCustomPath(e)}
```
Comment (taylorjdawson) [inline packages/common/src/views/AccountSelect.svelte:323]:
```suggestion
              on:change={handleCustomPath}
```

**Review by taylorjdawson — APPROVED** [review]
Approve with minor suggestions 🙌🏾

File: packages/common/src/views/AccountSelect.svelte [hunk 3]
```diff
@@ -263,18 +316,30 @@
...
-              >
-                {path.label} - {path.value}
+          {#if customDerivationPath}
+            <input type='text' 
+              class='base-path-select' 
+              placeholder="type/your/custom/path..." 
+              on:change={handleCustomPath}
+            /> 
+            <span class='input-select' on:click={toggleDerivationPathToDropdown}></span>
+          {:else if  !customDerivationPath}
```
Comment (aaronbarnardsound) [inline packages/common/src/views/AccountSelect.svelte:326]: Is this missing parenthesis?

File: packages/common/src/views/AccountSelect.svelte [hunk 4]
```diff
@@ -263,18 +316,30 @@
...
-          <select
-            class='base-path-select'
-            bind:value={scanAccountOptions['derivationPath']}
-          >
-            {#each basePaths as path, pathIndex}
-              <option
-                value={path.value}
-              >
-                {path.label} - {path.value}
+          {#if customDerivationPath}
```
Comment (aaronbarnardsound) [inline packages/common/src/views/AccountSelect.svelte:319]: Missing parenthesis?

**Review by aaronbarnardsound — APPROVED** [review]

File: packages/common/src/views/AccountSelect.svelte (hunk 3 above)
Comment (Adamj1232) [reply packages/common/src/views/AccountSelect.svelte:326]: custom derivation path is a boolean, so just checking the truthy-ness of it here...is that what you are referring to?

**Pull request merged**